demo/README.md

```diff
@@ -0,0 +1,6 @@
+### Smart Sync Demo
+This demo syncs the state of a contract in Polygon to Goerli
+
+
+## TODO
+-Modularize RelayContract to take as input an instance of DendrETH. This contract will return the validated block roots. 
```

Comment:
@RafaelAPB 
Are we using DendrETH now or still smart-sync?

Reply:
We will be using DendrETH, but we can include it at the end